src/components/ColumnFilter/ColumnFilter.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { SearchLink } from '../SearchLink/SearchLink';
+import { useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  columnName: string;
+};
+
+export const ColumnFilter: React.FC<Props> = ({ columnName }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+  const formatedColumnName = columnName.toLocaleLowerCase();
+
+  const params: { sort: string | null; order: string | null } = {
+    sort: formatedColumnName,
+    order: sort === formatedColumnName ? 'desc' : null,
```

Comment:
The logic for setting `order` to 'desc' might not be correct. Currently, it sets `order` to 'desc' if `sort` equals `formatedColumnName`, but it should also consider the current `order` state. Consider revising the logic to toggle between 'asc' and 'desc' based on the current `order` value.